server/src/models/cat/cat.model.ts

```diff
@@ -0,0 +1,28 @@
+import { Table, Model, Column, DataType, PrimaryKey, AutoIncrement, HasMany } from 'sequelize-typescript';
+import { Mouse } from '../mouse/mouse.model';
+
+@Table({
+  tableName: 'cats'
+})
+    
+export class Cat extends Model<Cat> {
+  @PrimaryKey
+  @AutoIncrement
+  @Column(DataType.INTEGER)
+  id: number;
```

Comment:
create a base model give it the common columns
all the models will have an id its too bad to add it to each one
in addition to id its a good practice to add deletedAt & createdAt a common practice
i would also change id to uuid generally a numerical id is more performant but recent tests show each negligent in addition a numerical id can cause security risks which you don't have with uuid